packages/backend/src/open_payments/quote/routes.ts

```diff
@@ -54,14 +54,16 @@ async function createQuote(
   ctx: CreateContext<CreateBody>
 ): Promise<void> {
   const { body } = ctx.request
+  const options: CreateQuoteOptions = {
+    paymentPointerId: ctx.paymentPointer.id,
+    receiver: body.receiver,
+    clientId: ctx.grant?.clientId
+  }
+  if (body.sendAmount) options.sendAmount = parseAmount(body.sendAmount)
+  if (body.receiveAmount)
+    options.receiveAmount = parseAmount(body.receiveAmount)
   try {
-    const quoteOrErr = await deps.quoteService.create({
-      paymentPointerId: ctx.paymentPointer.id,
-      receiver: body.receiver,
-      sendAmount: body.sendAmount && parseAmount(body.sendAmount),
-      receiveAmount: body.receiveAmount && parseAmount(body.receiveAmount),
-      clientId: ctx.grant?.clientId
-    })
```

Comment:
I _think_ you should be able to make `sendAmount` and `receiveAmount` mutually exclusive in `CreateBody` (like with `CreateQuoteOptions`) and leave this as it was before.
This is enforced by the OpenAPI request validator thanks to:
- https://github.com/interledger/open-payments/pull/230

Reply:
With the changes to the `CreateBody` type, it's still complaining:
`Types of property 'sendAmount' are incompatible. Type 'Amount' is not assignable to type 'never'`

But I think we should keep the changes to `CreateBody`.